rhobs/alerting/data_plane/prometheus.availability_metric_alerts.yaml

```
@@ -0,0 +1,45 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: konflux-up-alerting
```

```suggestion
  name: rhtap-konflux-up-alerting
```
or
```suggestion
  name: rhtap-up-alerting
```
Our alerting rules still need to be prefixed with `rhtap-` to pass the CI pipeline. The first option seems slightly less terrible to me.